lib/licensee/project_files/package_manager_file.rb

```diff
@@ -10,21 +10,26 @@ class PackageManagerFile < Licensee::ProjectFiles::ProjectFile
 
       # Hash of Filename => [possible matchers]
       FILENAMES_EXTENSIONS = {
-        'DESCRIPTION' => [Matchers::Cran],
-        'dist.ini'    => [Matchers::DistZilla]
+        'DESCRIPTION'  => [Matchers::Cran],
+        'dist.ini'     => [Matchers::DistZilla],
+        'LICENSE.spdx' => [Matchers::Spdx]
+      }.freeze
+
+      FILENAMES_SCORES = {
+        'package.json' => 1.0,
+        'LICENSE.spdx' => 1.0,
```

Comment:
@wking this and `name_score` below are only used to prioritize which files to look at, see https://github.com/benbalter/licensee/blob/389c22e7e16378ed57c47b7b89e22e95281f8dab/lib/licensee/projects/project.rb#L84 they are not the confidence assigned to a license statement found in a package manager file. That's at https://github.com/benbalter/licensee/blob/389c22e7e16378ed57c47b7b89e22e95281f8dab/lib/licensee/matchers/package.rb#L14 (which specifies 90)